pipeline/minimmit.md

```diff
@@ -212,13 +214,17 @@ _If you have already broadcast `notarize(c, v)` for a `c` that cannot be finaliz
 - Replicas enter `v + 1` as soon as they see some `proof(v)` (as fast as `L` messages). If the network is partitioned in two, replicas in each half of the partition may continue to enter successive views (on different `proof(v)`s) but will never finalize conflicting blocks. To bound the depth of forks in a partition, replicas can wait to enter some view `v + k` until they have seen `Q` messages in view `v`.
 - A Byzantine leader could equivocate, sending a distinct proposal to each replica and causing them to broadcast a `notarize(*, v)` for different blocks. After a replica observes `≥ L` `notarize(*, v)` messages for some `* != c`, it will then choose to broadcast a `nullify(v)` message. Eventually, `L` `nullify(v)` messages will be received and honest replicas will enter `v + 1` (within `Δ` of the first honest replica).
 - Since at most `f` nodes are Byzantine or faulty, once an honest leader is assigned, it is possible for at least `Q` correct replicas to finalize a block (including all of its ancestors).
+- A faulty leader can delay the network by at most `3Δ`. If replicas time out at `2Δ`, then a `nullification(v)` will be seen by replicas by time `3Δ`.
+- A Byzantine leader can delay the network by at most `4Δ`. If the leader equivocates with multiple proposals such that no single proposal can reach a quorum of `L`, and such that no `nullification(v)` can be produced in a single round of voting, then replicas who previously sent a `notarize(c, v)` message must send a second `nullify(v)` message. Since all original messages are sent within `2Δ` of entering the round, the second round of messages are sent within `3Δ`. This means the second round of messages are received by other replicas within `4Δ`, allowing them to generate a `nullification(v)`.
```

Comment:
nit: "are sent within `2Δ` of entering the round" (who entering?)

I think we can again use the `t0` here to make this easier to reason about.